README.md

```diff
@@ -0,0 +1,106 @@
+![linkerd2][logo]
+
+[![Build Status][ci-badge]][ci]
+[![GitHub license][license-badge]](LICENSE)
+[![Slack Status][slack-badge]][slack]
+
+This repo contains the transparent proxy component of [Linkerd2][linkerd2].
+While the Linkerd2 proxy is heavily influenced by [the Linkerd 1.X
+proxy][linkerd1], it comprises an entirely new codebase implemented in the
+[Rust programming language][rust].
+
+This proxy's features include:
+
+* Transparent, zero-config proxying for HTTP, HTTP/2, and arbitrary TCP protocols.
+* Automatic [Prometheus][prom] metrics export for HTTP and TCP traffic;
+* Transparent, zero-config WebSocket proxying;
+* Opportunistic TLS;
+* [P2C + Peak-EWMA][loadbalancing] HTTP load balancing; and
+* An on-demand diagnostic `tap` API.
+
+This proxy is primarily intended to run on Linux in containerized
+environments like [Kubernetes][k8s], though it may also work on other
+Unix-like systems (like MacOS).
+
+The proxy supports service discovery via the [`Destination` gRPC
+service][linkerd2-proxy-api] and DNS.
```

Comment:
I would say "The proxy supports service discovery via DNS and via the linkerd2 `Destination` gRPC API" or similar.